pysam/libcbcf.pyi

```diff
@@ -150,7 +150,7 @@ class VariantHeader:
         contig: Optional[str] = ...,
         start: int = ...,
         stop: int = ...,
-        alleles: Optional[Tuple[str]] = ...,
+        alleles: Optional[Tuple[str, ...]] = ...,
```

Comment:
I believe all of these are variable length, so need an ellipsis after: https://docs.python.org/3/library/typing.html#typing.Tuple